src/app/app.routing.ts

```diff
@@ -35,6 +36,12 @@ export const CLIENT_ROUTER_PROVIDERS = [AuthGuard];
 
 const APP_ROUTES: Routes = [
   { path: '', component: HomeComponent, pathMatch: 'full', data: { title: 'Dockstore' } },
+  {
```

Comment:
If you do the other change I suggested, you should be able to get rid of this entirely. Then you toggle the new dashboard with `http://localhost:4200?newDashboard` question mark, not slash.